README.md

```diff
@@ -11,3 +11,15 @@ $ npm start
 ```
 
 Open your browser to ***http://localhost:8080*** to view.  Modify your React code in your editor, save your changes, and watch those changes automatically reflect in the browser! *(the power of webpack-dev-server)*# f
+
```

Comment:
Document your team's git workflow by copying _CONTRIBUTING.md into the root directory of your repo and editing it suite your needs. It follows a forking workflow.

just looking through the learn app.  I think this stuff should be moved under a contribution guide.